src/commands/dbsize.json

```diff
@@ -12,6 +12,10 @@
         ],
         "acl_categories": [
             "KEYSPACE"
+        ],
+        "command_tips": [
+            "REQUEST_POLICY:ALL_MASTERS",
```

Comment:
Is it too much to be the PC police and ask we do "ALL_PRIMARIES" >_<

Reply:
i prefer not introducing new terms. AFAIK, nothing in redis uses the term Primary, so using it in my eyes just adds confusion.
i think using the term Master here is just another drop in the sea, but if you feel strongly that we should avoid it, do you think Shards is good in this context? is it used in redis cluster in other places in similar context?

Reply:
had a chat with Madelyn, decided to use `shards` and `nodes` just to avoid triggering anyone with this addition..